public/app/plugins/datasource/prometheus/querybuilder/shared/operationUtils.ts

```diff
@@ -23,6 +23,90 @@ export function functionRendererRight(model: QueryBuilderOperation, def: QueryBu
   return str + params.join(', ') + ')';
 }
 
+export function rangeRendererRightWithParams(
```

Comment:
This would probably benefit a bit of doc comment, explaining why there is a Right/Left version of this as it is not apparent right away.

Reply:
Or maybe this could be deduplicated a bit? Looking at the code it seems the only difference is a single `params.unshift` vs `params.push` call.

Reply:
Hmmm... good idea. Let me see...